### DISCUSSION TITLE
Ownership of ytnef

### DISCUSSION THREAD

**Discussion by Yeraze** [Yeraze/ytnef#99] [General]
Starting this up so that we can discuss the elephant in the room, Ownership of this tool.

@ohwgiles , I know you've worked on it some over the years so I'll flagging you specifically.

I don't use this tool any more.  In fact, I haven't actively used it in about 15 years.  I try to merge PR's and do minor maintenance as time allows, but for a tool I don't actually use on a daily basis anymore it's pretty low priority. It keeps getting picked up by other tools (Evolution, Geary, etc) so it needs someone who can maintain it long term.

Thought? Volunteers?

**Comment by ohwgiles** [comment]
I got into this because I use Geary, and a few years ago I used to receive a lot of unreadable `winmail.dat` attachments. So I'm responsible for Geary's dependency on this library, and have taken an interest in the CVEs because they potentially affect me (alongside every other user of Geary or Evolution).

Until `winmail.dat` goes away forever (which isn't likely to happen soon), this library will remain an invisible but very useful dependency for mail clients.

I'm very stretched for free time currently, so I couldn't commit to doing more than what you anyway do. If I did end up with time to spend on TNEF I think I would rather spend it on a new (maybe Rust) library.

I understand it's low priority for you but I'm happy with that situation. If it's untenable for you, I can step in, but don't expect great things...

**Reply by Yeraze** [reply]
No problem, I just want to make it available in case someone out there is willing & able to take it over... I'm happy to keep merging PR's and making releases, but when it comes to actual debugging and such my time is limited (and frankly, my exposure since I don't even use this tool any longer)
